src/wallet/index.ts

```diff
@@ -145,15 +150,45 @@ class Wallet {
    * Signs a transaction offline.
    *
    * @param transaction - A transaction to be signed offline.
-   * @param options - Options to include for signing.
+   * @param multisignAddress - Multisign only. An account address corresponding to the multi-signature being added. If this
+   * wallet represents your [master keypair](https://xrpl.org/cryptographic-keys.html#master-key-pair) you can get your account address
+   * with the Wallet.getClassicAddress() function.
    * @returns A signed transaction.
    */
   public signTransaction(
     transaction: Transaction,
-    options: SignOptions = { signAs: '' },
+    multisignAddress: string = '',
```

Comment:
```suggestion
    multisignAddress?: string,
```
This shouldn't be an empty string if it's not needed, it should be undefined